department-of-veterans-affairs/vets-website
# [CST] - Update CST V1 Components to V3 Components PR #1

## Summary

Updated ClaimContentionList.jsx to use uswds

Updated ClaimDetailLayout.jsx to use uswds

Updated ClaimLetterListItem.jsx to use uswds

Updated src/applications/claims-status/constants.js so that it had CST_BREADCRUMB_BASE that is used in ClaimsBreadcrumbs.jsx

Updated ClaimsBreadcrumbs.jsx to use uswds and updated the following components that used ClaimBreadcrumbs
- src/applications/claims-status/components/ClaimDetailLayout.jsx
- src/applications/claims-status/containers/AppealInfo.jsx
- src/applications/claims-status/containers/AskVAPage.jsx
- src/applications/claims-status/containers/ClaimEstimationPage.jsx
- src/applications/claims-status/containers/DocumentRequestPage.jsx
- src/applications/claims-status/containers/StemClaimStatusPage.jsx
- src/applications/claims-status/containers/YourClaimLetters/index.jsx

Updated tests for the following:
- src/applications/claims-status/tests/e2e/page-objects/TrackClaimsPage.js
- src/applications/claims-status/tests/components/appeals-v2/AppealInfo.unit.spec.jsx
- src/applications/claims-status/tests/components/ClaimsBreadcrumbs.unit.spec.jsx
- src/applications/claims-status/tests/e2e/08.claim-letters-keyboard-only.cypress.spec.js
- src/applications/claims-status/tests/e2e/08.claim-letters.cypress.spec.js

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#77095

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| ClaimContentionList.jsx  | ![Screenshot 2024-03-22 at 3 25 23 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/141954992/a361d251-07bf-4f51-8324-2aea8a94092d) | ![Screenshot 2024-03-22 at 2 29 06 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/141954992/c4ac238a-8cbf-4823-b9e4-787351d35621) |
| ClaimDetailLayout.jsx |        |       |
| ClaimLetterListItem.jsx | ![Screenshot 2024-03-21 at 3 35 50 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/141954992/b6944918-69d5-475b-96bd-858a4c5a9bc4) | ![Screenshot 2024-03-21 at 3 35 01 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/141954992/10717536-8272-4b64-8697-abb6bd03b0fc) |
| ClaimsBreadcrumbs.jsx | ![Screenshot 2024-03-21 at 3 48 21 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/141954992/80c53895-4d37-4973-96bd-5b941674cdf0) | ![Screenshot 2024-03-22 at 3 23 59 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/141954992/3d5dc291-c795-488b-9848-cb2dc4f37403) |

## What areas of the site does it impact?

Claim Status Tool

## How to Test

- `ClaimLetterListItem.jsx`, go to http://localhost:3001/track-claims/your-claim-letters and check `va-link`
- 

### Quality Assurance & Testing


### Error Handling


### Authentication


### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:
